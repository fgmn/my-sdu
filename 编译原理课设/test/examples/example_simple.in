CONST C=0;
VAR A,B;
BEGIN
	READ(B,A);
	A:=B+C;
	WRITE(A);
END.