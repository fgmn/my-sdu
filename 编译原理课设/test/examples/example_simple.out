PROGRAM(SUBPROG(CONSTANTDECLARE(CONST,CONSTANTDEFINE(C,=,0),;),VARIABLEDECLARE(VAR,A,COMMA,B,;),SENTENCE(COMBINED(BEGIN,SENTENCE(READSENTENCE(READ,LP,B,COMMA,A,RP)),;,SENTENCE(ASSIGNMENT(A,:=,EXPRESSION(ITEM(FACTOR(B)),+,ITEM(FACTOR(C))))),;,SENTENCE(WRITESENTENCE(WRITE,LP,A,RP)),;,SENTENCE(EMPTY),END))),.)
